src/test/java/com/google/sps/UserTest.java

```diff
@@ -22,10 +22,13 @@
 import com.google.maps.model.LatLng;
 import com.google.maps.model.LocationType;
 import java.util.logging.Logger;
+import io.github.cdimascio.dotenv.Dotenv;
```

Comment:
I'm not sure how to go about doing this, but we might want to look into the licensing for this to make sure that we're not violating any open source policy.

Reply:
It has a MIT license and I believe that should be okay for us to use

Reply:
The license permitting commercial use is here